language/German/strings.po

```diff
@@ -261,7 +261,7 @@ msgstr "Schieberegler"
 
 #: src/emu/ui/custui.cpp:541
 msgid "Gfx viewer background"
-msgstr "Hintergrund Gfx Anzeige"
+msgstr "Hintergrund Gfx-Anzeige"
```

Comment:
I have a few more corrections... shall I wait until this is merged into master? Or send by PM?